doc/source/selections.rst

```diff
@@ -7,11 +7,13 @@ Atom selection language
 
 AtomGroups can be created by selecting atoms using the MDAnalysis atom selection language:
 
-  >>> from MDAnalysis.tests.datafiles import PSF
-  >>> u = mda.Universe(PSF)
-  >>> ala = u.select_atoms('resname ALA')
-  >>> ala
-  <AtomGroup with 190 atoms>
+.. ipython:: python
+
+    import MDAnalysis as mda
+    from MDAnalysis.tests.datafiles import PSF, DCD
+    u = mda.Universe(PSF, DCD)
+    ala = u.select_atoms('resname ALA')
+    ala
 
 The :meth:`~MDAnalysis.core.groups.AtomGroup.select_atoms` method of a
```

Comment:
I’d also include a line clarifying that select atoms operates on the contents of the Group it was called on. So u.select_atoms selects from all, but on an AtomGroup it selects from within that group